agones/src/xds.rs

```diff
@@ -57,6 +57,7 @@ mod tests {
     /// for this test, we should only have single Agones integration test in this suite, since they
     /// could easily collide with each other.
     async fn agones_token_router() {
+        quilkin::test_utils::enable_log("agones=debug");
```

Comment:
Depending on what aspect you want to debug, you will nee to also include a call to `debug_pods(client: &Client, labels: String)` to actually get logs and events about those pods.

For example, `debug_pods(client, "role=proxy")` will output debug information about the proxy instances. You can see the labels they have here:

https://github.com/googleforgames/quilkin/blob/c0d5ca5c1052366a097959711e6c75b2634befbb/agones/src/xds.rs#L374

(This would be good to add to the doc on debug_pods) 

https://github.com/googleforgames/quilkin/blob/d8e93b60ab0be42d8ae7b020d21302345d6dfe7b/agones/src/lib.rs#L418-L425 )

Reply:
Also looking at the build log, it's not failing because something failed to come up (as far as I can tell anyway), it's failing here:

https://github.com/googleforgames/quilkin/blob/c0d5ca5c1052366a097959711e6c75b2634befbb/agones/src/xds.rs#L201

So the existing `debug_pod` statements aren't going to help in this moment - they have to be conditionally put in.